compiler/codegen/symbol.cpp

```diff
@@ -1281,11 +1302,17 @@ void FnSymbol::codegenDef() {
         INT_FATAL("LLVM function verification failed");
       }
     }
+
+    if(llvmFuncOptDump == 0 && strcmp(llvmFuncDumpName, name) == 0)
```

Comment:
I think it would be nice if 0 meant no dumping, to save some strcmparing in the common case